yt_dlp/extractor/common.py

```diff
@@ -151,6 +151,11 @@ class InfoExtractor:
                                    for HDS - URL of the F4M manifest,
                                    for DASH - URL of the MPD manifest,
                                    for MSS - URL of the ISM manifest.
+                    * manifest_data
+                                The media playlist m3u8 data as a string,
```

Comment:
for consistency with above
```suggestion
                                The M3U8 media playlist data as a string,
```